src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const wall = document.querySelector(`.wall`);
+  const spider = wall.querySelector('.spider');
```

Comment:
Consider adding a check to ensure that both 'wall' and 'spider' elements exist before proceeding. If either element is not found, attempting to access their properties will result in a runtime error.